prusti-server/src/backend.rs

```diff
@@ -36,6 +38,10 @@ impl<'a> Backend<'a> {
                     viper.verify(viper_program)
                 })
             }
+            Backend::Lithium(lithium) => {
+                Stopwatch::start("prusti-server", "vir verification");
```

Comment:
Without this, the stopwatch is immediately dropped and does *not* measure how long the verification takes.
```suggestion
                let _stopwatch = Stopwatch::start("prusti-server", "vir verification");
```